plaso/lib/lexer.py

```diff
@@ -12,6 +12,8 @@
 
 from plaso.lib import errors
 
+# pylint: disable=missing-type-doc,missing-return-type-doc,missing-return-doc
```

Comment:
Please make these local overrides instead of overrides for the whole module (repeat every else)

Reply:
Done